README.md

```diff
@@ -503,6 +503,8 @@ To start your Phoenix server:
       ```elixir
       :crypto.strong_rand_bytes(32) |> :base64.encode
       ```
+    - ALLOW_API_ORIGINS: origins allowed to request the application API
```

Comment:
@SimonLab  this is a good idea in the short-term. 👍
(_we'll need a way of allowing any origin based on an API key in medium term_)